cellarium/cas/client.py

```diff
@@ -240,6 +263,12 @@ def print_user_quota(self) -> None:
             f"Weekly quota reset date: {user_quota['quota_reset_date']}\n"
             f"Lifetime quota: {lifetime_quota}, Remaining lifetime quota: {remaining_lifetime_quota} "
         )
+        if (
+            not user_quota["quota_increased"]
+            and user_quota["remaining_lifetime_quota"] is not None
+            and user_quota["remaining_lifetime_quota"] < 25000
```

Comment:
Please put 25000 in constants and call it something like LOW_QUOTA_WARNING_THRESHOLD.

Reply:
Ooo good catch.  Forgot to do that